custom/abdm/milestone_one/utils/response_handler.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework.response import Response
```

Comment:
I think request_util was a good naming above. response_handler sounds like its handling response but it does not look like to be getting/generating responses, so `response_util` sounds better on same lines.

Reply:
Renamed